ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -66,25 +66,25 @@ open class SegmentedControl: UIControl {
         func segmentTextColorSelected(for window: UIWindow) -> UIColor {
             switch self {
             case .tabs:
-                return UIColor(light: Colors.primary(for: window), dark: .white)
+                return UIColor(light: Colors.primary(for: window), dark: Colors.textDominant)
             case .switch:
-                return UIColor(light: Colors.primary(for: window), dark: Colors.gray100)
+                return UIColor(light: Colors.primary(for: window), dark: Colors.textDominant)
```

Comment:
making sure this is intentional, since we went from white/gray100 to the same color

Reply:
yes it is. the Figma file has already reflected the change.